### PR TITLE
Refactor-2/5: Move transpiled files to dist/ folder

### PR DESCRIPTION
**Pull request by ilia-kebets-sonarsource**

### CODE REVIEW

File: CreateCardForIssue/action.yml [hunk 1]
```diff
@@ -9,4 +9,4 @@ inputs:
     required: true
 runs:
   using: 'node16'
-  main: 'CreateCardForIssue.js'
+  main: '../dist/CreateCardForIssue/CreateCardForIssue.js'
```
Comment (pavel-mikula-sonarsource) [inline CreateCardForIssue/action.yml:12]: This change also needs to be done for `AssignCardToSender` directory

**Review by pavel-mikula-sonarsource — CHANGES_REQUESTED** [review]
One more line needs to be updated

File: CreateCardForIssue/action.yml (hunk 1 above)
Comment (ilia-kebets-sonarsource) [reply CreateCardForIssue/action.yml:12]: done https://github.com/SonarSource/gh-action-lt-backlog/pull/19/commits/76f32b1cb07512ff0d912e47028c63df377277d1

**Review by pavel-mikula-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**